lya_likelihood/py/likelihood.py

```diff
@@ -103,6 +113,14 @@ def __init__(self,data=None,theory=None,emulator=None,
     def set_free_parameters(self,free_parameter_names,free_param_limits):
         """Setup likelihood parameters that we want to vary"""
 
+        # WE SHOULD DOCUMENT THIS FUNCTION BETTER. WHEN WOULD WE USE THIS?
+        # ALSO, SHOULD RENAME free_parameters TO free_param_names
+        # AND IT IS NOT CLEAR WHY WE NEED TO STORE EITHER THAT OR EVEN
+        # free_param_limits IN THE LIKELIHOOD OBJECT.
+        # ISN'T IT ENOUGH TO STORE free_params?
+        # ALSO, WHY DO WE NEED TO PASS THE ARGUMENTS ABOVE, IF THEY ARE
+        # ALREADY STORED?
+
```

Comment:
@Chris-Pedersen - I left these comments here, but we could address them on a different branch, after having a chat (may be next week?).